tests/deployment/vlan_networks.py

```diff
@@ -146,7 +155,7 @@ def delete_networks():
             # Assert that none of the nodes are on any network
             port_networks = self.get_port_networks(ports)
             for node in nodes:
-                assert self.get_network(node.nics[0].port, port_networks) == set()
+                assert self.get_network(node.nics[0].port, port_networks) == set()  # noqa
```

Comment:
Maybe we should just change `port_networks` to a slightly shorter name? I agree it's more readable on one line.